MichaelKintscher/qualitative-coding-app
# Sets timestamp width dependent slider margins

Before this patch, the sliders that make up the scalable scrubbing bar occupied as much vertical space as possible in the horizontal layouts they were placed in. In order to fit timestamp labels on the extreme ends of a slider, or in their future tick-mark bars, we need extra space on their left and right sides. This patch adds margins, based on the maximum width of a timestamp label, to the sides of each slider.

Testing Steps:
  1. Run the application
  2. Verify that the slider labels are left aligned
  3. Verify that each slider has the same left and right margins
  4. Verify that the timestamp labels in the scaling bar are not cut off
  5. Load a video
  6. Verify that the functionality of the sliders work as expected

Type: New Feature
